views/index.html

```diff
@@ -4,9 +4,18 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+  <script src="./main.js"></script>
```

Comment:
Another best practice concept (but it does depend on what your needs are so this may not be relevant) is to try and always load your scripts at the bottom of your body. The intention for this is to allow all the html to load before the script loads, making your page appear more responsive. This rule may not apply in this case, but if you can live without immediately calling the scripts, it is better to default to having them at the bottom. Feel free to reply with "\@lumodon" if you wanted to discuss further.